pkg/apiaccesskeys/apiaccesskey.go

```diff
@@ -0,0 +1,103 @@
+package apiaccesskeys
+
+// APIAccessKeyBase represents a New Relic API access key.
+type APIAccessKeyBase struct {
+	ID    string `json:"id,omitempty"`
+	Key   string `json:"key,omitempty"`
+	Name  string `json:"name,omitempty"`
+	Notes string `json:"notes,omitempty"`
+	Type  string `json:"type,omitempty"`
+}
+
+// APIAccessIngestKey represents a New Relic API access ingest key.
+type APIAccessIngestKey struct {
+	APIAccessKeyBase
+	AccountID  int    `json:"accountId,omitempty"`
+	IngestType string `json:"ingestType,omitempty"`
+}
+
+// APIAccessUserKey represents a New Relic API access user key.
+type APIAccessUserKey struct {
+	APIAccessKeyBase
+	AccountID int    `json:"accountId,omitempty"`
+	UserID    string `json:"userId,omitempty"`
+}
+
+type apiAccessIngestKeyCreateResponse struct {
+	APIAccessKeys struct {
+		CreatedKeys []APIAccessIngestKey `json:"createdKeys"`
```

Comment:
One thing we should consider is that the create mutation facilitates creating **ingest keys** _and_ **user keys**. We might want to think about adjusting this type definition to allow that.

e.g. `CreatedKeys []APIAccessKey` -- this could return both user keys and ingest keys.